komga-webui/src/locales/en.json

```diff
@@ -51,24 +51,24 @@
     "button_scan": "Scan",
     "button_select_series": "Select Series",
     "field_import_path": "Import from folder",
-    "info_part1": "This screen lets you import files that are outside your existing libraries. You can only import files into existing Series, in which case Komga will move or copy the files into the directory of the chosen Series.",
-    "info_part2": "If you choose a number for a book, and a book already exists with that number, then you will be able to compare the 2 books. If you decide to import the book, Komga will upgrade the existing book with the new one, effectively replacing the old file with the new.",
+    "info_part1": "Here you can import files into a selected series of your existing libraries.",
+    "info_part2": "Choose a number for a book and compare the two if it exists. Importing the book upgrades the entry and replaces the old file.",
     "no_files_found": "No files found",
     "notification": {
       "go_to_book": "Go to book",
-      "import_failure": "Failed to import book: {file}",
-      "import_successful": "Book imported successfully: {book}",
```

Comment:
"Successfully" does nothing other than imply importing books in a manner that succeeds is out of the ordinary, or, that books can be imported in an unsuccessful fashion. Neither is (should be) true.